File: official/docs/java/v5/billing/delete.java
package billing;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.Billing;
import com.easypost.model.PaymentMethod;

public class Delete {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = "EASYPOST_API_KEY";

        boolean success = Billing.deletePaymentMethod(PaymentMethod.Priority.PRIMARY);

        System.out.println(success);
    }
}

File: official/docs/csharp/v4/billing/delete.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            bool success = await client.Billing.DeletePaymentMethod(PaymentMethod.Priority.Primary);

            Console.WriteLine(success.ToString());
        }
    }
}
